WooCommerce/Classes/POS/Presentation/PointOfSaleCardPresentPaymentAlert.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+struct PointOfSaleCardPresentPaymentAlert: View {
+    private let alertType: PointOfSaleCardPresentPaymentAlertType
+
+    init(alertType: PointOfSaleCardPresentPaymentAlertType) {
+        self.alertType = alertType
+    }
+
+    var body: some View {
+        switch alertType {
+        case .scanningForReaders(let alertViewModel):
+            PointOfSaleCardPresentPaymentScanningForReadersView(viewModel: alertViewModel)
+        case .scanningFailed(let alertViewModel):
+            PointOfSaleCardPresentPaymentScanningForReadersFailedView(viewModel: alertViewModel)
+        case .bluetoothRequired(let alertViewModel):
+            PointOfSaleCardPresentPaymentBluetoothRequiredAlertView(viewModel: alertViewModel)
+        case .foundReader(let alertViewModel):
+            PointOfSaleCardPresentPaymentFoundReadersView(viewModel: alertViewModel)
+        case .updatingReader(let alertViewModel):
+            CardPresentPaymentReaderUpdateInProgressView(viewModel: alertViewModel)
+        case .updateFailed(let alertViewModel):
+            CardPresentPaymentReaderUpdateFailedView(viewModel: alertViewModel)
```

Comment:
nit: any reasons these two views aren't renamed to `PointOfSale*` like the views in other cases?